app/views/reservations/_reservation.html.erb

```diff
@@ -55,6 +48,8 @@
     <% end %>    
   <% end %>
   <% if reservation.review.nil? && reservation.finished? && Review.left_review?(reservation.home_id, current_user) %>
```

Comment:
Personally, I don't recommend leaving controller classes in views:

1. Because you overload the view
2. Because you leave the use of the controller accessible.

I recommend you look for an alternative, such as using a helper, to avoid placing the class.

Reply:
Thanks for the recommendation, I already used a "helper" to avoid use the controller, and also simplify the condition